lighthouse-core/gather/computed/trace-of-tab.js

```diff
@@ -83,17 +87,19 @@ class TraceOfTab extends ComputedArtifact {
     }
 
     // subset all trace events to just our tab's process (incl threads other than main)
-    const processEvents = trace.traceEvents.filter(e => {
-      return e.pid === startedInPageEvt.pid;
-    }).sort((event0, event1) => event0.ts - event1.ts);
+    const processEvents = trace.traceEvents
+      .filter(e => {
+        return e.pid === startedInPageEvt.pid;
+      })
+      .sort((event0, event1) => event0.ts - event1.ts);
 
     return {
       processEvents,
       startedInPageEvt: startedInPageEvt,
       navigationStartEvt: navigationStart,
       firstPaintEvt: firstPaint,
       firstContentfulPaintEvt: firstContentfulPaint,
-      firstMeaningfulPaintEvt: firstMeaningfulPaint
+      firstMeaningfulPaintEvt: firstMeaningfulPaint,
```

Comment:
I started using https://github.com/prettier/prettier on demand in this PR. I'm already in love.